app/models/public_body/calculated_home_page.rb

```diff
@@ -1,5 +1,27 @@
 # Guess the home page based on the request email domain.
 module PublicBody::CalculatedHomePage
+  extend ActiveSupport::Concern
+
+  included do
+    cattr_accessor :excluded_calculated_home_page_domains, default: %w[
+      aol.com
+      btinternet.com
+      gmail.com
```

Comment:
should also add googlemail.com before merge

Reply:
```suggestion
      gmail.com
      googlemail.com
```